IO/GeoFileParser.cpp

```diff
@@ -717,12 +717,13 @@ Goal * GeoFileParser::parseWaitingAreaNode(TiXmlElement * e)
         if(int value = xmltoi(attribute, -1); value > -1 && attribute == std::to_string(value)) {
             wa->SetRoomID(value);
         } else {
-            Log->Write("ERROR:\t  waiting area %d: room_id set but not an integer", wa->GetId());
+            Logging::Error(
+                fmt::format("waiting area {:d}: room_id set but not an integer", wa->GetId()));
```

Comment:
Sorry, but please add `check_fmt` Macro on format string to check the Format string at compile-time (see [documentation](https://fmt.dev/latest/index.html#safety)).
`check_fmt` itself is defined in `general/Format.h`. Please also use this include instead of including `<fmt::format.h>`.
```suggestion
                fmt::format(check_fmt("waiting area {:d}: room_id set but not an integer"), wa->GetId()));
```